invenio_grobid/tasks.py

```diff
@@ -24,12 +24,12 @@
 
 """Custom exception types."""
 
-import invenio_records.api
+from celery import shared_task
 
-from invenio.celery import celery
+from invenio_records import Record
 
 
-@celery.task
+@shared_task(ignore_result=True)
 def upload(results):
     """Upload record using `invenio_records.api.create_record`."""
-    return invenio_records.api.create_record(results)
+    return Record.create(results)
```

Comment:
Meaning that `GROBID_RESULT_HANDLER` will be None, and then just fix the api.submit_records to handle that case.